packages/sitecore-jss-nextjs/src/services/base-graphql-sitemap-service.ts

```diff
@@ -258,12 +259,12 @@ export abstract class BaseGraphQLSitemapService {
 
       aggregatedPaths.push(formatPath(item.path));
 
-      // check for type safety's sake - personalize may be empty depending on query type
-      if (item.route?.personalization?.variantIds.length) {
+      const variantIds = item.route?.personalization?.variantIds?.filter(
+        (variantId) => !variantId.includes('_') // exclude component A/B test variants
```

Comment:
Will AB testing not work with SSG?

Reply:
Correct, we are not pre-rendering all permutations of component tests on a page (since there is technically no limit to the number of tested components and their variations on a page - there are obvious performance impacts). This will be a documented limitation.